sdk/core/azure-core/src/main/java/com/azure/core/util/ExpandableStringEnum.java

```diff
@@ -28,10 +28,7 @@
      * <p>
      * This constructor shouldn't be called as it will produce a {@link ExpandableStringEnum} which doesn't
      * have a String enum value.
-     *
-     * @deprecated Use the {@link #fromString(String, Class)} factory method.
      */
-    @Deprecated
```

Comment:
We should still keep the deprecated annotation here as this ctor should not be used.